pset-functions.js

```diff
@@ -90,7 +102,20 @@ Use the following test cases to confirm your program meets the success criteria
 console.log('Problem 2:')
 
 // Add your code below this line
+function wordCount(phrase) {
+  const wordPhrase = phrase.replace(/[,]/g, '')
+  const wordCountObject = {}
+
+  wordPhrase.split(' ').forEach(function (currentValue) {
+    wordCountObject[currentValue] = wordCountObject[currentValue] ? ++wordCountObject[currentValue] : 1
+  })
+
+  return wordCountObject
+}
 
+const result = wordCount("Humpty Dumpty sat on a wall Humpty Dumpty had a great fall")
+
+console.log(result)
 // Add your code above this line
```

Comment:
@OcielC Good job here putting together this solution. Good use of chaining the split and forEach methods and using the conditional ternary operator to either add the new word to the object with the value of 1 or find the word in the object if it exists and increment the value by 1.

Take a look at an alternate solution [here](https://github.com/jsr20190826/hw-02-functions-solution/blob/a4d789161886ff115657133c279062df8a4ca72d/pset-functions.js#L107)